src/dotenv/cli.py

```diff
@@ -125,6 +124,32 @@ def run(ctx: click.Context, override: bool, commandline: List[str]) -> None:
     ret = run_command(commandline, dotenv_as_dict)
     exit(ret)
 
+@cli.command()
+@click.pass_context
+def example_file(ctx: click.Context) -> None:
```

Comment:
You can use `generate_sample` instead of `example_file`. It will be a more explicit verb and closer to the action performed.